src/examples/rotation.app.json

```diff
@@ -0,0 +1,579 @@
+{
+    "metadata": {},
```

Comment:
1. Do we really need to save this? Could we dynamically generate the scene based on the content of the Thres.js canvas? 
2. In case we pass a different material structure, will the content be re-generated based on it?

Reply:
1. Yes we can, this scene is just an example.
2. No, it depends only on the content that exists in that scene.